cmake.deps/cmake/UnibiliumCMakeLists.txt

```diff
@@ -7,16 +7,18 @@ set_target_properties(unibilium PROPERTIES PUBLIC_HEADER ${PROJECT_SOURCE_DIR}/u
   VERSION "${VERSION_MAJOR}.${VERSION_MINOR}")
 
 if(NOT WIN32)
-  execute_process(COMMAND "shell ncursesw6-config --terminfo-dirs 2>/dev/null || \
+  execute_process(COMMAND sh -c "ncursesw6-config --terminfo-dirs 2>/dev/null || \
```

Comment:
is `shell` some sort of cmake magic?

Reply:
It's nonsense, it's not a thing and the above doesn't work. Likely a Makefile-ism. We didn't notice this because we don't use the the cmakelists on unix. This has already been fixed upstreams in neovim/unibilium.